assets/i18n/diq.json

```diff
@@ -73,7 +73,7 @@
     "I18N_INTERACTIONS_SET_INPUT_EMPTY_SET": "kHer satır rê jew wesiqe dek)",
     "I18N_INTERACTIONS_SET_INPUT_NO_ANSWER": "cewab nêdeya.",
     "I18N_INTERACTIONS_SET_INPUT_SUBMIT": "Bırışe",
-    "I18N_LANGUAGE_FOOTER_VIEW_IN": "Oppa'y ena zıwan dı bıasnê:",
+    "I18N_LANGUAGE_FOOTER_VIEW_IN": "Oppai'y ena zıwan dı bımotné:",
```

Comment:
That's a good catch. I don't know enough of this language to tell, though -- I've seen Oppiay elsewhere in that file as well.

Maybe we should just take this in as-is? If there's a mistake, it will get fixed on translatewiki at some point and we can pull the updates from there then.